openshift/templates/dps-payment-service.dc.yaml

```diff
@@ -0,0 +1,116 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: dps-payment-service-deployment-config
+  creationTimestamp: null
+parameters:
+  - name: APP_NAME
+    displayName: Name
+    description: A prefix appended to all objects
+    required: true
+  - name: SOURCE_CONTEXT_DIR
+    required: true
+  - name: GIT_URL
+    required: true
+  - name: REPO_REF
+    required: true
+  - name: PROJECT_PREFIX
+    required: true
+  - name: SPRING_PROFILES_ACTIVE
+    required: true
+    value: "cloud,splunk"
+  # TODO value needs to change at some point
+  - name: BAMBORA_PAYMENT_ENDPOINT
+    required: true
+    value: "https://example.com"
```

Comment:
This should come from openshift configuration

Reply:
Agreed - once we get there.  We're going for a 'hello world' first and incremental population as we progress.

Reply:
this has been resolved, I've removed the value and we can set as a secret later.